projects/swimlane/ngx-ui/package.json

```diff
@@ -45,9 +45,9 @@
     "json-schema": "*"
   },
   "dependencies": {
-    "@angular/flex-layout": "8.0.0-beta.26",
+    "@angular/flex-layout": "8.0.0-beta.27",
```

Comment:
Are we sure about this? I think it's safe to double check that the splitDirective is still working.

Reply:
Good catch. Updated the demo pages. Looks like it doesn't like using the directive name as an input:

`<div [ngxSplit]="'row'">`

but likes this:

`<div ngxSplit [ngxSplit]="row">`

Reply:
But that's probably Ivy-related, not flex-layout necessarily.